terraform/ec2/win/main.tf

```diff
@@ -190,12 +190,24 @@ resource "null_resource" "integration_test_run_validator" {
     destination = module.validator.instance_validator_config
   }
 
+  //runs validator and sets up prometheus java agent
   provisioner "remote-exec" {
     inline = [
+      "mkdir C:\\jmx_workload",
+      "powershell.exe -Command \"'---', 'rules:', '- pattern: \\\".*\\\"' | Set-Content -Path \\\"C:\\jmx_workload\\exporter_config.yaml\\\"\"",
+      "powershell.exe -Command \"'global:', '  scrape_interval: 1m', '  scrape_timeout: 10s', 'scrape_configs:', '  - job_name: jmx-exporter', '    sample_limit: 10000', '    file_sd_configs:', '      - files: [ \\\"C:\\\\jmx_workload\\\\prometheus_file_sd.yaml\\\" ]' | Set-Content -Path \\\"C:\\jmx_workload\\prometheus.yaml\\\"\"",
```

Comment:
Personally I'd rather write the YAML file into this test package, and then use powershell to copy the file over to where we want it.

Reply:
+1 . I would add a backlog item to track updates needed for this. This is adding a new pattern on top of the many ones we already have.

Reply:
totally agree. I would just have a powersehll script file (or other types) with this many commands that TF can write to a host then it's used by the test.